extensions/vertx-core/runtime/src/test/java/io/quarkus/vertx/core/runtime/VertxCoreProducerTest.java

```diff
@@ -61,6 +61,7 @@ public void shouldNotFailWithDefaultConfig() {
         configuration.workerPoolSize = 10;
         configuration.warningExceptionTime = Duration.ofSeconds(1);
         configuration.internalBlockingPoolSize = 5;
+        configuration.preferNativeTransport = false;
```

Comment:
No need to initialize as the default value is false anyway

Reply:
You're right.